torchvision/ops/poolers.py

```diff
@@ -258,3 +258,6 @@ def forward(
             result = _onnx_merge_levels(levels, tracing_results)
 
         return result
+
+    def __repr__(self) -> str:
+        return f"{self.__class__.__name__}(output_size={self.output_size}, sampling_ratio={self.sampling_ratio})"
```

Comment:
Maybe we can also add `featmap_names` as it is also a user's input ?

Reply:
I was wondering what to do about this: I left it out, since I believe the FPN does not display it as well. But happy to reconsider, if you think it's a good addition :ok_hand:

Reply:
I just added so that it's comprehensive on this `__repr__` at least

Reply:
Thanks !

> if you think it's a good addition

I have no strong opinion on that. Just thought that `featmap_names` is a configuration provided by user and repr displays 2 of 3 its configs. 

> since I believe the FPN does not display it as well

Seems like there is no repr at all for FPN, https://github.com/pytorch/vision/blob/master/torchvision/ops/feature_pyramid_network.py